analysed-packages/zlib/version-1.2.12/README.md

```diff
@@ -2,6 +2,10 @@
 
 https://zlib.net/fossils/zlib-1.2.12.tar.gz
 
+## Package URL (purl)
+
+pkg:none/zlib.net/fossils/zlib@1.2.12
```

Comment:
```suggestion
pkg:generic/zlib@1.2.12?download_url=https://zlib.net/fossils/zlib-1.2.12.tar.gz
```

Reply:
Thank you, considered your suggestions and adapted all other occurrences accordingly. Please see second attempt.